triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -199,6 +220,7 @@ object Server {
       //this).
       val runningTrigger = server.getRunningTrigger(uuid)
       runningTrigger.runner ! TriggerRunner.Stop
+      server.triggerStatus(runningTrigger, "stopped")
```

Comment:
Would be useful to specify that this case is an expected stop.
```suggestion
      server.triggerStatus(runningTrigger, "stopped: by user request")
```

Reply:
ok